File: Bouncy.py
NumBouncy = 0
NoBouncy = 99
x = 99


def NumeroBouncy(num):
    CadNum = str(num)
    CadNumOrdenado = ''.join(sorted(list(CadNum)))
    return not(CadNum == CadNumOrdenado or CadNum == CadNumOrdenado[::-1])


while float(NumBouncy)/(NoBouncy + NumBouncy) < 0.99:
    x += 1
    if NumeroBouncy(x):
        NumBouncy += 1
    else:
        NoBouncy += 1
print(x)

File: Bouncy.js
function Cre(Num){
    r = false;
    for (let index = 0; index < Num.length; index++) {
        if(Num[index] < Num[index + 1]){
            if(index + 1 == Num.length - 1){
                r = true;
                break;
            }
        }
    }
    return r;
}

console.log(Cre('123'))